source/mir/ndslice/selection.d

```diff
@@ -1556,18 +1557,28 @@ IndexSlice!N indexSlice(size_t N)(auto ref size_t[N] lengths)
 }
 
 ///
-@safe @nogc pure nothrow unittest
+@safe pure nothrow unittest
```

Comment:
I start to dislike  the Allocator now, but it might be worth to think about adding support for it?